### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by mfrister**
Comments welcome :)

### CODE REVIEW

**Comment by mugwump** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/528f35c8c7f3a33b7d03cde8/compare/feature/better_readme">analyzed this pull request</a>.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# OpenProject
+
+[<img src="https://travis-ci.org/opf/openproject.png?branch=feature/rails3" alt="Build Status" />](https://travis-ci.org/opf/openproject)
+[<img src="https://gemnasium.com/opf/openproject.png" alt="Dependency Status" />](https://gemnasium.com/opf/openproject)
+
+OpenProject is a web-based project management software. Its key features are:
+
+* [Work Package Tracking](https://www.openproject.org/projects/openproject/wiki/Work_packages)
+* [Project Timelines](https://www.openproject.org/projects/openproject/wiki/Timelines)
+* [A Wiki](https://www.openproject.org/projects/openproject/wiki/Wiki)
```
Comment (ulferts) [inline README.md]: Why is that "A Wiki" when it could simply be "Wiki"

Comment (mfrister) [inline README.md]: I somehow felt "Wiki" doesn't fit in with the other feature descriptions. I'll change it to "Wikis" to match the others.

**Comment by NobodysNightmare** [comment]
Not immediately related to this PR, but when I follow the link to the [Plugins project](https://www.openproject.org/projects/plugins) there is quite few visible content (especially in contrast to the other links provided).

There might be the need for a proper landing page...

File: README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+## Credits
+
+### Icons
+
+Thanks to Le Moign Vincent and his fabulous Minicons icons on [webalys.com](http://www.webalys.com/minicons/icons-free-pack.php).
+
+### Lato Font
+
+Thanks to tyPoland Lukasz Dziedzic for his 'Lato' font.
```
Comment (NobodysNightmare) [inline README.md]: I am quite sure that some error occured here...

Google gives me a _Lukasz Dziedzic_ as font author, but I guess the `tyPoland`-prefix was put there accidentially...

File: README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+### OpenProject Icon Font
+Published and created by the OpenProject Foundation (OPF) under [Creative Commons Attribution 3.0 Unported License](http://creativecommons.org/licenses/by/3.0/)
+with icons from the following sources
+[Minicons Free Vector Icons Pack](http://www.webalys.com/minicons) and
+[User Interface Design framework](http://www.webalys.com/design-interface-application-framework.php) both by webalys
+
+**Creative Commons License**
```
Comment (NobodysNightmare) [inline README.md:74]: This paragraph seems to duplicate the paragraph above, but I am not a lawyer.
However be aware of the inconsistent origin of the OpenProject Icon Font in this paragraph:
- OpenProject Icon Font **by OpenProject Team**
- a mention of the **OpenProject Foundation**

I'd prefer some consistency here...

File: README.md (hunk 2 above)
Comment (ulferts) [inline README.md]: Thats his nick. Not sure how one could write it in a clearer way. Maybe just some markup. Or remove the name altogether and add a link to the tyPoland string.

Comment (NobodysNightmare) [inline README.md]: Oh, I see...

some common conventions:
`Lukasz "tyPoland" Dziedzic`
`Lukasz Dziedzic (aka "tyPoland")`
`tyPoland (aka Lukasz Dziedzic)`

Adding a link is always a good idea :+1:

File: README.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+## License
+
+OpenProject is licensed under the terms of the GNU General Public License version 3.
+See [doc/COPYRIGHT.rdoc](doc/COPYRIGHT.rdoc) for details.
+
+## Credits
+
+### Icons
+
+Thanks to Le Moign Vincent and his fabulous Minicons icons on [webalys.com](http://www.webalys.com/minicons/icons-free-pack.php).
```
Comment (ulferts) [inline README.md]: That guys name is [Vincent Le Moign](https://plus.google.com/+VincentLeMoign/posts)

File: README.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+OpenProject is supported by its community members, both companies and individuals. There are different possibilities of getting help:
+
+* OpenProject [support page](https://www.openproject.org/support)
+* E-Mail Support - info@openproject.org
+
+## Contributing
+
+In case you find a bug or need a feature, please create a [new work package on openproject.org](https://www.openproject.org/projects/openproject/work_packages).
+
+If you want to contribute code, feel free to create a pull request on GitHub. Unfortunately we'll have to ask you to sign a [Contributor License Agreement](https://www.openproject.org/projects/openproject/wiki/Contributor_License_Agreement), but you can do that electronically so this shouldn't be too much pain.
```
Comment (ulferts) [inline README.md]: Maybe we should write something that sounds less scary. At least explain why we need it so people won't expect to loose their work and home afterwards.

Comment (mfrister) [inline README.md]: I actually don't know why we need it. Do you have a proposal for a less scary sentence?

File: README.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+* [Project Timelines](https://www.openproject.org/projects/openproject/wiki/Timelines)
+* [Wikis](https://www.openproject.org/projects/openproject/wiki/Wiki)
+* [Forums](https://www.openproject.org/projects/openproject/wiki/Forums)
+
+Via plugins, it also supports:
+
+* [Meeting Management](https://www.openproject.org/projects/openproject/wiki/Meetings)
+* [Scrum Process Support](https://www.openproject.org/projects/openproject/wiki/Agile_teams)
+* [Time and Cost Reporting](https://www.openproject.org/projects/openproject/wiki/Time_and_Cost)
+* and [others](https://www.openproject.org/projects/plugins).
```
Comment (hschink) [inline README.md]: I agree with @NobodysNightmare: The plug-in overview is not really helpful. A wiki page with a list of plug-ins and a short description would be more appropriate I guess. Feel free to ask me whether I could do that. :smile:

Comment (mfrister) [inline README.md]: @hschink Could you do that? :smiley_cat:

File: README.md (hunk 2 above)
Comment (mfrister) [inline README.md]: Chose the second form.

Any ideas where to link to?

File: README.md (hunk 3 above)
Comment (mfrister) [inline README.md:74]: Changed to _OpenProject Foundation (OPF)_.

File: README.md (hunk 4 above)
Comment (mfrister) [inline README.md]: Changed.

File: README.md (hunk 6 above)
Comment (hschink) [inline README.md]: @meeee Here you are: [Plug-In Overview](https://www.openproject.org/projects/openproject/wiki/OpenProject_Plug-Ins)

Comment (mfrister) [inline README.md]: Changed, thanks!

**Pull request merged**